Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/util/ConnectionHelper.java

```diff
@@ -143,4 +144,33 @@ protected PropertyAccessor getPropertyAccessor() {
         return PropertyAccessor.getInstance();
     }
 
+    public Map<String, String> getOrgNameRemoteHcidExternalEntities() {
+        HashMap<String, String> organizationMap = new HashMap<>();
+        List<BusinessEntity> externalEntityList = new ArrayList<>(getExternalEntitiesMap().values());
+        if (NullChecker.isNotNullish(externalEntityList)) {
+            for (BusinessEntity businessEntity : externalEntityList) {
+                List<KeyedReference> keyedReference = getKeyedReference(businessEntity);
+                if (NullChecker.isNotNullish(keyedReference)) {
+                    organizationMap.put(getEntityName(businessEntity), keyedReference.get(0).getKeyValue());
+                }
+            }
+        }
+        return organizationMap;
+    }
+
+    public Map<String, String> getRemoteHcidOrgNameMap() {
+        HashMap<String, String> organizationMap = new HashMap<>();
+        List<BusinessEntity> entities = getAllBusinessEntities();
+        if (NullChecker.isNotNullish(entities)) {
+            for (BusinessEntity businessEntity : entities) {
+                List<KeyedReference> keyedReference = getKeyedReference(businessEntity);
+                if (NullChecker.isNotNullish(keyedReference)
+                    && NullChecker.isNotNullishIgnoreSpace(keyedReference.get(0).getKeyValue())) {
+                    organizationMap.put(HomeCommunityMap.getHomeCommunityWithoutPrefix(
+                        keyedReference.get(0).getKeyValue()), getEntityName(businessEntity));
+                }
+            }
+        }
+        return organizationMap;
```

Comment:
It can be one method without having "urn:oid:" prefix for RemoteHcid in the dropdown and the same can be passed  on to service layer. we don't need 2 maps.